packages/sentinel_one/data_stream/threat/elasticsearch/ingest_pipeline/default.yml

```
@@ -28,6 +22,24 @@ processors:
         - json.id 
       target_field: _id
       ignore_missing: true
+  - script:
+      description: Set the value of event.category and event.type.
+      lang: painless
+      source: >
+          def eventCategory = new ArrayList();
+          def eventType = new ArrayList();
+          if (ctx.json?.threatInfo?.threatId != null && ctx.json?.threatInfo?.threatId != '') {
+              def threat_classification = ctx.json?.threatInfo?.classification;
+              if (threat_classification == 'Exploit' || threat_classification == 'PUA') {
+                  eventCategory.add('threat');
+                  eventType.add('indicator');
+              } else if (threat_classification == 'Malware' || threat_classification == 'Ransomware' || threat_classification == 'Trojan' || threat_classification == 'Downloader') {
```

```suggestion
              } else if ['Malware', 'Ransomware', 'Trojan', 'Downloader'].contains(threat_classification)) {
```
Maybe similar for above? I have no strong opinion either way for that though.